src/main/java/com/exadel/ipromise/controllers/addictionController.java

```diff
@@ -0,0 +1,25 @@
+package com.exadel.ipromise.controllers;
+
+import com.exadel.ipromise.dto.AddictionListDto;
+import com.exadel.ipromise.service.AddictionService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin(origins = "http://localhost:8080")
+@RequestMapping("/addiction")
+public class addictionController {
```

Comment:
name of class always must stats from capital character

Reply:
Done